prov/util/src/util_cntr.c

```diff
@@ -313,3 +313,28 @@ int ofi_cntr_init(const struct fi_provider *prov, struct fid_domain *domain,
 	return 0;
 }
 
+ofi_ep_cntr_inc_func ofi_ep_tx_cntr_inc_funcs[] = {
+	[ofi_op_msg] = ofi_ep_tx_cntr_inc,
+	[ofi_op_tagged] = ofi_ep_tx_cntr_inc,
+	[ofi_op_read_req] = ofi_ep_rd_cntr_inc,
+	[ofi_op_read_rsp] = ofi_ep_rd_cntr_inc,
```

Comment:
And after all that discussion...  my natural inclination is to see rem_rd for a tx read_rsp.

Reply:
Sure, make sense. We'll just have to make sure these are used in the correct way. My original interpretation for tx/rx cntr functions was that they were being called by the initiator/target, respectively, of the original op, not necessarily the internal op.